### ISSUE TITLE
Explicit feedback explanation

### ISSUE THREAD

**Issue by mcginty** [dragonman225/stm32f469-usbaudio-example#8]
Hi there! I'm trying to understand the feedback mechanism, and this code seems to have a number of different formulae in comments, and I'm not sure if these are just magic numbers you got to through experimentation or if they're from some constants about the hardware.

It seems like the current formula can be simplified as
$$\frac{ deviationFromNominal^3 * 128 }{912673}$$

given the code:
```
fb_value += audio_buf_writable_size_dev_from_nom * audio_buf_writable_size_dev_from_nom / 912673 * 128 * audio_buf_writable_size_dev_from_nom;
```

**Comment by dragonman225** [comment]
These are magic numbers. The way I did feedback was by monitoring the remaining capacity of the audio buffer array, and telling the host to send more/less data in future packets, to ensure no buffer overrun/underrun happen.

This is not the conventional way to do feedback. The standard way is to set up a timer that monitors the I2S output buffer, measuring the exact feedback value, as defined in [USB Specification 5.10.4.2](http://esd.cs.ucr.edu/webres/usb11.pdf). But I was too lazy to set up and debug another piece of peripheral device, so I came up with this formula that mimicked the behavior.

Btw I guess the reason I didn't combine the multiplication into $deviationFromNomial^3$ was because that could overflow and make the calculation wrong.
